proto/api.go

```diff
@@ -87,27 +87,28 @@ func (s stringSet) keys() []string {
 
 // AllMethods specifies the complete set of methods.
 var AllMethods = stringSet{
-	Contains:              struct{}{},
-	Get:                   struct{}{},
-	Put:                   struct{}{},
-	ConditionalPut:        struct{}{},
-	Increment:             struct{}{},
-	Delete:                struct{}{},
-	DeleteRange:           struct{}{},
-	Scan:                  struct{}{},
-	EndTransaction:        struct{}{},
-	ReapQueue:             struct{}{},
-	EnqueueUpdate:         struct{}{},
-	EnqueueMessage:        struct{}{},
-	AdminSplit:            struct{}{},
-	AdminMerge:            struct{}{},
-	Batch:                 struct{}{},
-	InternalHeartbeatTxn:  struct{}{},
-	InternalGC:            struct{}{},
-	InternalPushTxn:       struct{}{},
-	InternalResolveIntent: struct{}{},
-	InternalMerge:         struct{}{},
-	InternalTruncateLog:   struct{}{},
+	Contains:               struct{}{},
```

Comment:
No. The `struct{}` can be removed, but `gofmt -s` is not smart enough to do it automatically in this case.